### PR TITLE
Include instructions for `bun` package manager

### PR DESCRIPTION
**Pull request by colinhacks**
### Improving Documentation

- [x] Run `pnpm prettier-fix` to fix formatting issues before opening the PR.
- [x] Read the Docs Contribution Guide to ensure your contribution follows the docs guidelines: https://nextjs.org/docs/community/contribution-guide

### What?

Add instructions for using `bun/bunx` where relevant. I only added mentions where npm/yarn/pnpm were all already listed. 

### Why

Bun can be used as a runtime-agnostic [package manager](https://bun.sh/package-manager) and script runner in any project with a `package.json`.

(Sorry, I probably should have consolidated this with https://github.com/vercel/next.js/pull/53467)

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 68064c14ff83582a99208add7dd5e33f9ae57301

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 68064c14ff83582a99208add7dd5e33f9ae57301

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 68064c14ff83582a99208add7dd5e33f9ae57301

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/08-next-cli.mdx [hunk 1]
```diff
@@ -146,7 +146,7 @@ To learn more about Telemetry, [please read this document](https://nextjs.org/te
 ## Next Info
 
 `next info` prints relevant details about the current system which can be used to report Next.js bugs.
-This information includes Operating System platform/arch/version, Binaries (Node.js, npm, Yarn, pnpm) and npm package versions (`next`, `react`, `react-dom`).
+This information includes Operating System platform/arch/version, Binaries (Node.js, npm, Yarn, pnpm, Bun) and npm package versions (`next`, `react`, `react-dom`).
```
Comment (styfle) [inline docs/02-app/02-api-reference/08-next-cli.mdx:149]: Your PR didn't change the behavior of `next info` so this isn't true

Comment (colinhacks) [reply docs/02-app/02-api-reference/08-next-cli.mdx]: Apologies, fixed

**Review by styfle — APPROVED** [review]

**Review by ijjk — APPROVED** [review]

**Pull request merged**